### ISSUE TITLE
OMUSE wrapper for DALES

### ISSUE THREAD

**Issue by goord** [NLeSC/TEAM2018#14]
In the Cloud-resolving modeling project we have planning to write a software paper on the OMUSE interface for the Dutch Atmospheric Large Eddy Simulation (DALES). This MPI-parallel Fortran code is exposed as a python object through this interface and can be manipulated programmatically and dynamically from within e.g. a Jupyter Notebook. This should make setting up test cases for the program much easier and enable the application of external forcings on the system. It would be nice to actually demonstrate the value of this by re-creating dynamically forced test cases, such as the cold air outbreak, from within a python script.

**Comment by romulogoncalves** [comment]
For the 4 days of the sprint the people to who this issue is assigned should report hours on:
CloudResolving (@goord what is the project code?)

Only report the days you were present at the Sprint. If you switched teams, report that days in the project specified in the issue of that team.

For extra work you need to request time to the tech leads by sending an email to techleads@esciencecenter.nl

**Comment by romulogoncalves** [comment]
@goord once the paper is submitted, please let us through a comment on this issue and close the issue.

**Comment by romulogoncalves** [comment]
@goord could you let us know if the paper was submitted?
Thanks.

**Comment by goord** [comment]
Hi Romulo,
The paper is not yet submitted, I still need a couple of days to work on it.

**Comment by romulogoncalves** [comment]
@goord were all the goals achieved? Could you summarize in 3 lines what was achieved? Can we consider it done and close this issue?

**Comment by goord** [comment]
The software has not been released yet and the paper not submitted, but good progress has been made: 
- Paper draft has been written, but still needs code examples and review from project partners
- Python API was made more user-friendly and OMUSE-compliant
- New functionality has been added to the wrapper, enabling the user to nudge the model through our interface.

I will close the issue after releasing the software and submitting of the manuscript.

**Comment by romulogoncalves** [comment]
@goord what is the current status?

**Comment by goord** [comment]
@romulogoncalves Python API is being made more complete, I am testing it on all known supported test cases of the model. The manuscript is nearly finished, we need to add code snippets and list the repositories involved. This work is being continued on project budget and is part of the eScience output.
